tests/run/test_fstring.pyx

```diff
@@ -1234,11 +1234,9 @@ non-important content
         # spec of '=10'.
         self.assertEqual(f'{x:=10}', '        20')
 
-        """
```

Comment:
There's a few above in `__test_debug_conversion` to enable once https://github.com/cython/cython/pull/3939/files is in.

Reply:
Pr got merged, might need to rebase and enable the tests :)